packages/browser-wallet/src/popup/pages/Account/Tokens/Tokens.tsx

```diff
@@ -138,7 +138,6 @@ function Tokens() {
 export default function TokensRoutes() {
     const account = useSelectedCredential();
     const nav = useNavigate();
-    useAtomValue(tokensAtom); // Ensure tokens are kept in memory
```

Comment:
This is not needed after all?

Reply:
Nope.. I had misunderstood something about how atoms work. Thought they were removed from memory when they unmount, but that doesn't seem to be the case.